packages/roosterjs-editor-dom/lib/utils/Browser.ts

```diff
@@ -6,9 +6,10 @@ const isAndroidRegex = /android/i;
  * Get current browser information from user agent string
  * @param userAgent The userAgent string of a browser
  * @param appVersion The appVersion string of a browser
+ * @param vendor The vendor string of a browser
  * @returns The BrowserInfo object calculated from the given userAgent and appVersion
  */
-export function getBrowserInfo(userAgent: string, appVersion: string): BrowserInfo {
+export function getBrowserInfo(userAgent: string, appVersion: string, vendor: string): BrowserInfo {

```

Comment:
Please make the new parameter optional to be compatible with older version

Reply:
Done